server/handlers/logic.go

```diff
@@ -0,0 +1,89 @@
+package handlers
```

Comment:
Agreed. Just didn't want to get hung up on a filename which is irrelevant in the context of Go. Happy to change it to anything else but I haven't come up with a good name for it :-/